src/PrestaShopBundle/Controller/Admin/Sell/Catalog/Product/ProductController.php

```diff
@@ -67,6 +75,51 @@ class ProductController extends FrameworkBundleAdminController
      */
     private const PRODUCT_CONTROLLER_PERMISSION = 'ADMINPRODUCTS_';
 
+    /**
+     * Shows products listing.
+     *
+     * @AdminSecurity("is_granted(['read'], request.get('_legacy_controller'))")
+     *
+     * @param Request $request
+     * @param ProductFilters $filters
+     *
+     * @return Response
+     */
+    public function indexAction(Request $request, ProductFilters $filters): Response
+    {
+        $productGridFactory = $this->get('prestashop.core.grid.factory.product');
+        $productGrid = $productGridFactory->getGrid($filters);
+
+        return $this->render('@PrestaShop/Admin/Sell/Catalog/Product/index.html.twig', [
+            'productGrid' => $this->presentGrid($productGrid),
+            'enableSidebar' => true,
+            'layoutHeaderToolbarBtn' => $this->getProductToolbarButtons(),
+            'help_link' => $this->generateSidebarLink('AdminProducts'),
+        ]);
+    }
+
+    /**
+     * Provides filters functionality
+     *
+     * @AdminSecurity("is_granted('read', request.get('_legacy_controller'))")
+     *
+     * @param Request $request
+     *
+     * @return RedirectResponse
+     */
+    public function searchAction(Request $request): RedirectResponse
```

Comment:
Since the search action is a regular one here, you can use the Common controller here with just routing configuration https://devdocs.prestashop.com/1.7/development/components/grid/tutorials/work-with-search-form/#common-search-controller

Reply:
I see, so yeah I can just safely delete this. Thanks!